examples/celery-rabbitmq/README.md

```diff
@@ -187,39 +183,36 @@ The question now is, how do you see what's going on?
 
 Flower is a web-based tool for monitoring and administrating Celery clusters. By connecting to the node that contains Celery, you can see the behaviour of all the workers and their tasks in real-time.
 
-To bring up the frontend, run this command `$ kubectl create -f examples/celery-rabbitmq/celery-controller.json`. This controller is defined as so:
-
-```js
-{
-  "id": "flower-controller",
-  "kind": "ReplicationController",
-  "apiVersion": "v1beta1",
-  "desiredState": {
-    "replicas": 1,
-    "replicaSelector": {"name": "flower"},
-    "podTemplate": {
-      "desiredState": {
-        "manifest": {
-          "version": "v1beta1",
-          "id": "flower",
-          "containers": [{
-            "name": "flower",
-            "image": "endocode/flower",
-            "cpu": 100,
-            "ports": [{"containerPort": 5555, "hostPort": 5555}]
-          }]
-        }
-      },
-      "labels": {
-        "name": "flower",
-        "app": "taskQueue"
-      }
-    }
-  },
-  "labels": {
-    "name": "flower"
-  }
-}
+To bring up the frontend, run this command `$ kubectl create -f examples/celery-rabbitmq/flower-controller.yaml`. This controller is defined as so:
+
+```yaml
+apiVersion: v1beta3
+kind: ReplicationController
+metadata:
+  labels:
+    name: flower
+  name: flower-controller
+spec:
+  replicas: 1
+  selector:
+    name: flower
+  template:
+    metadata:
+      labels:
+        app: taskQueue
+        name: flower
+    spec:
+      containers:
+      - capabilities: {}
```

Comment:
How are you connecting to the flower frontend?  

What cloud provider / distro / networking setup are you using?

If it's just a control/status dashboard, you could try our proxy mechanism in the apiserver:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/accessing-the-cluster.md

In general, we intend services to be used for accessing applications, however:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services

It's ok to leave hostPort here for now, but please add a comment that explains why it's there and a TODO to switch to one of these other mechanisms.